source/module_hamilt_general/module_xc/xc_functional_libxc.cpp

```diff
@@ -153,16 +153,70 @@ std::vector<xc_func_type> XC_Functional_Libxc::init_func(const std::vector<int>
 		else if( id == XC_GGA_X_ITYH ) // short-range of B88_X
 		{
 			add_func( XC_GGA_X_ITYH );
-			double parameter_omega[1] = {PARAM.inp.exx_hse_omega}; // GlobalC::exx_info.info_global.hse_omega
+			double parameter_omega[1] = {GlobalC::exx_info.info_global.hse_omega}; // GlobalC::exx_info.info_global.hse_omega

```

Comment:
Please don't use GlobalC

Reply:
This follows the original coding conventions. Replacing GlobalC would entail substantial refactoring of the existing framework in [input_conv.cpp ](https://github.com/deepmodeling/abacus-develop/blob/develop/source/module_io/input_conv.cpp), which is beyond the intended scope of this PR

Reply:
You can use PARAM instead of GlobalC

Reply:
Deleted